fastchat/serve/monitor/clean_battle_data.py

```diff
@@ -36,6 +36,7 @@
     "google",
     "llama",
     "NETWORK ERROR DUE TO HIGH TRAFFIC. PLEASE REGENERATE OR REFRESH THIS PAGE.",
+    "$MODERATION$",
```

Comment:
Use the whole sentence to reduce false postive.
`$MODERATION$ YOUR INPUT VIOLATES OUR CONTENT MODERATION GUIDELINES.`